pyonfx/convert.py

```diff
@@ -16,12 +16,15 @@
 # along with this program. If not, see http://www.gnu.org/licenses/.
 
 from __future__ import annotations
+from fractions import Fraction
 import re
 import math
 import colorsys
 from enum import Enum
+import string
```

Comment:
Why is this import needed?

Reply:
We don't need it. I forget to delete it

Reply:
We don't need it. I forget to delete it.